apps/system-e2e/src/tests/islandis/consultation-portal/smoke/consultation-portal-authenticated.spec.ts

```diff
@@ -30,7 +32,26 @@ test.describe.skip('Consultation portal authenticated', () => {
 
   test('front page should have expected static content', async () => {
     const page = await context.newPage()
-    await page.goto('/samradsgatt')
+    await page.goto(URL)
```

Comment:
Please consider this recommendation I just added to the docs - https://www.notion.so/System-E2E-testing-workshop-29c089c182844933b992015c4ef24791?pvs=4#ac866522e289425ca5dfe992c883a8ac

Reply:
icelandicAndNoPopupUrl function used for goto(URL)